static/docs/commands-reference/import.md

```diff
@@ -72,3 +73,45 @@ force updating the downloaded data artifact from the external DVC repo.
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example
+
+An obvious case for this command is to import a dataset from an external DVC
+repo, such as our
+[get started example repo](https://github.com/iterative/example-get-started).
+
+```dvc
+$ dvc import git@github.com:iterative/example-get-started data/data.xml
+Importing 'data/data.xml (git@github.com:iterative/example-get-started)' -> 'data.xml'
+...
+Saving information to 'data.xml.dvc'.
+...
+```
+
+In contrast with `dvc get`, this command doesn't just download the data file,
+but it also creates an import stage (DVC-file) to keep track of this <abbr>data
+artifact</abbr> as a special `repo`
+[external dependency](/doc/user-guide/external-dependencies). Check
+`data.xml.dvc`:
+
+```yaml
+md5: 7de90e7de7b432ad972095bc1f2ec0f8
+wdir: .
+locked: true
+deps:
+- path: data/data.xml
+  repo:
```

Comment:
we should update `https://dvc.org/doc/user-guide/dvc-file-format` btw :)

Reply:
Good point. Updated in 710b95c, do you think that's enough of a change?

Reply:
yep, should be enough for now